.ci/scripts/linux-test.sh

```diff
@@ -20,5 +20,10 @@ OUTPUT_DIR="$(pwd)/build"
 OUTPUT_JSON_FILE="$OUTPUT_DIR/TEST-go-system_tests.out.json"
 OUTPUT_JUNIT_FILE="$OUTPUT_DIR/TEST-go-system_tests.xml"
 
+# Download systemtest dependencies soo the download time doesn't count towards
+# the test timeout.
+APM_ROOT=$(pwd)
+cd systemtest && go mod download && cd ${APM_ROOT}
```

Comment:
another way to do the same
```suggestion
cd systemtest && go mod download && cd -
```